extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -0,0 +1,318 @@
+|:----------------------------|:-----------|
+| `offset`                    | ✅        |
+| `rotation`                  | ✅        |
+| `scale`                     | ✅        |
+| `texCoord`                  | ❌        |
+
+#### [KHR_xmp_json_ld](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_xmp_json_ld)
+❌ No animatable properties.
+
+### Example target templates for extensions
+
+| `pointer`                                                                                       | Accessor Type | Component Type(s)   | Description              |
+|:------------------------------------------------------------------------------------------------|:--------------|:--------------------|:-------------------------|
+| `"/materials/{}/pbrMetallicRoughness/baseColorTexture/extensions/KHR_texture_transform/offset"` | `"VEC2"`      | `5126`&nbsp;(FLOAT) | XY offset vector         |
+| `"/extensions/KHR_lights_punctual/lights/{}/intensity"`                                         | `"SCALAR"`    | `5126`&nbsp;(FLOAT) | Light intensity          |
+| `"/materials/{}/extensions/KHR_materials_emissive_strength/emissiveStrength"`                   | `"SCALAR"`    | `5126`&nbsp;(FLOAT) | Emissive strength factor |
+
+## glTF Schema Updates
+
+* **JSON schema**: [glTF.KHR_animation_pointer.schema.json](schema/glTF.KHR_animation_pointer.schema.json)
+
+## Notes and clarifications
+
+_This section is non-normative._
+
+### JSON Pointer
+
+For a `SCALAR` value, the JSON Pointer targets the glTF property.  
+This means, the property is replaced by the interpolated value.
+  
+For any other case, the JSON Pointer targets the glTF property as well but **must** be an `array` property. For clarification, it is not allowed to animate a single element in an `array`.  
+This means, that the two or more elements do replace the values in the array.  
+
+### Material Animation
+
+Material animation targets materials directly. This means that when animating the material, all nodes using that material are updated together. If individual material animation per node is desired, each of them needs to have its own material to animate.
+
+### Mesh Animation
+
+Mesh animation targets meshes directly. This means that when animating the mesh, all nodes using that mesh are updated together. This is the case, when the `weights` are animated.
+
+## Examples
+
+The following snippet shows the changes for [`animations`](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/AnimatedCube/glTF/AnimatedCube.gltf#L117) from the [AnimatedCube](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedCube) asset.
+
+```javascript
+"animations" : [
+    {
+        "channels" : [
+            {
+                "name" : "Targeting x, y, z, w for a rotation of node at index 0."
+                "sampler" : 0,
+                "target" : {
+                    "path" : "pointer"
+                    "extensions": {
```

Comment:
This example JSON is marked as the wrong language, is not valid JSON (missing commas), and has inconsistent formatting.

```suggestion
```json
"animations": [
    {
        "channels": [
            {
                "name": "Targeting x, y, z, w for a rotation of node at index 0.",
                "sampler": 0,
                "target": {
                    "path": "pointer",
                    "extensions": {
```